uaa_client/authentication.py

```diff
@@ -154,7 +154,11 @@ def authenticate(self, request, uaa_oauth2_code=None, **kwargs):
         if access_token is None:
             return None
 
-        user_info = jwt.decode(access_token, verify=False)
+        user_info = jwt.decode(
+            access_token,
+            options={"verify_signature": False},
+            algorithms=["HS256", "RS256"],
```

Comment:
One question: how did you determine which algorithms to use for verifying the JWT? I'm looking at the UAA documentation and having a bit of trouble determining this myself

Reply:
Hey @markdboyd, those were the defaults and I was trying to replicate the current syntax. My apologies for not getting to this on Friday. Did you get confirmation that this was correct?

Reply:
Yeah we confirmed that both of those algorithms are supported by Cloud Foundry UAA from this code

https://github.com/cloudfoundry/uaa/blob/develop/server/src/main/java/org/cloudfoundry/identity/uaa/oauth/KeyInfo.java

Reply:
Got it! Thank you again 👍